debezium-storage/debezium-storage-redis/src/main/java/io/debezium/storage/redis/history/RedisSchemaHistory.java

```diff
@@ -87,6 +85,20 @@ public class RedisSchemaHistory extends AbstractSchemaHistory {
             .withDescription("Socket timeout (in ms)")
             .withDefault(DEFAULT_SOCKET_TIMEOUT);
 
+    private static final Field PROP_WAIT_ENABLED = Field.create(CONFIGURATION_FIELD_PREFIX_STRING + "wait.enabled")
+            .withDescription(
+                    "Enables wait for replica. In case Redis is configured with a replica shard, this allows to verify that the data has been written to the replica.")
+            .withDefault(false);
+    private static final Field PROP_WAIT_TIMEOUT = Field.create(CONFIGURATION_FIELD_PREFIX_STRING + "wait.timeout.ms")
+            .withDescription("Timeout when wait for replica")
+            .withDefault(1000L);
+    private static final Field PROP_WAIT_RETRY_ENABLED = Field.create(CONFIGURATION_FIELD_PREFIX_STRING + "wait.retry.enabled")
+            .withDescription("Enables retry on wait for replica failure")
+            .withDefault(false);
+    private static final Field PROP_WAIT_RETRY_DELAY = Field.create(CONFIGURATION_FIELD_PREFIX_STRING + "wait.retry.delay.ms")
+            .withDescription("Delay of retry on wait for replica failure")
+            .withDefault(1000L);
```

Comment:
can we share this as RedisOffsetBackingStore has the same field....

Reply:
they don't have the same prefix

Reply:
OK, in the adoc we describe only the properties of the sink. Where are those of the offset and schema history in case or Redis?

Reply:
There are  none. I did not find it for all the other older properties.

Reply:
@jpechane please, advise

Reply:
We could specify this `debezium.source.database.history.redis.address`, for example in case of the sink is not Redis (if it is, it will use the same credentials by default, unless we specify them explicitly to work against another Redis, for example).

Reply:
I will see how to fit all the properties in the documentation.